packages/@ember/-internals/runtime/lib/mixins/container_proxy.ts

```diff
@@ -2,7 +2,16 @@ import { schedule, join } from '@ember/runloop';
 /**
 @module ember
 */
+import Container, { FactoryManager } from '@ember/-internals/container/lib/container';
+import { TypeOptions } from '@ember/-internals/container/lib/registry';
 import { Mixin } from '@ember/-internals/metal';
+import { Factory } from '@ember/-internals/owner';
+
+export interface IContainer {
```

Comment:
For the future, it'd be great if we had a comment here explaining why we're differentiating this from `Container` as `IContainer`, how they related, and what the purpose of this particular interface is.